dockstore-webservice/src/main/java/io/dockstore/webservice/core/CollectionEntry.java

```diff
@@ -4,12 +4,14 @@
 import java.util.Date;
 
 import io.dockstore.common.SourceControl;
+import io.swagger.annotations.ApiModelProperty;
```

Comment:
Question: what happens in the openapi models right now? Are they auto-detected as long or as date-time strings?

Reply:
Also, surprised there is no swagger.yaml file update

Reply:
the openapi models are date-time strings (not good), but we don't really use any openapi annotations.

as for the swagger.yaml file, delaying as long as possible to push due to merge issues